pages/03.installation/01.environment/04.lando/docs.md

```diff
@@ -0,0 +1,188 @@
+---
+---
+title: Lando
+metadata:
+  description: Lando is a wrapper around Docker that simplifies the process for PHP applications to run on Docker.
+taxonomy:
+  category: docs
+---
+
+>>> To contribute to this documentation, please submit a pull request to our [learn repository](https://github.com/userfrosting/learn/tree/master/pages).
+
+Lando provides a stable and easy-to-use local development environment. It allows you to integrate PHPMyAdmin and most PHP IDE debuggers hook simply into this as well.
+
+>> Note: Lando is not meant for production!
+
+>> Note: Lando works best with Mac & Linux, but does support Windows as well. Note that if you are using Windows, you must enable Hyper-V. Doing so prevents you from running Vagrant/VMBox. If you are able to run Docker on Windows, you _should_ be able to run Lando on Windows as well.
+
+## Installation Steps
+
+### Install & Configure Lando
+
+Start by installing [Lando](https://docs.lando.dev/basics/installation.html#system-requirements).
+
+Next, create your Lando configuraton file in your application root. Call the file `.lando.yml`. A sample configuration is provided below:
+
+#### Sample Lando Configuration File
+
+```yml
+name: userfrosting
+recipe: lamp
+config:
+  webroot: ./public
+  php: '7.3'
+  database: mariadb
+  xdebug: true
+  cache: redis
+
+services:
+  appserver:
+    composer:
+      phpunit/phpunit: '*'
+    build_as_root:
+      - apt-get update -y
+      - apt-get install -my wget gnupg
+      - curl -sL https://deb.nodesource.com/setup_12.x | bash - # install node + (UF4.2.0 requires 10.12+) https://github
+      - apt-get install -y nodejs
+      - a2enmod headers
+    #config:
+    #  conf: ./lando-config/php.ini # uncomment this section to enable your custom php.ini file
+    overrides:
+      environment:
+        PHP_IDE_CONFIG: "serverName=userfrosting.lndo.site"
+    ssl: true
+
+
+  # Add a phpmyadmin db frontend
+  pma:
+
+    # Use the latest version of phpmyadmin
+    type: phpmyadmin
+
+    # The databases you want to look at, this will default to a service called
+    # "database"
+    #
+    # You might want to run `lando info` on your app to see what databases you
+    # have available
+    hosts:
+      - database
+
+# Add in a proxy route to phpmyadmin
+proxy:
+  pma:
+    - pma.userfrosting.lndo.site
+
+tooling:
+  phpunit:
+    service: appserver
+    description: "Run PHP Unit tests: lando phpunit"
+  composer:
```

Comment:
Based on https://docs.lando.dev/config/lamp.html#tooling, this should be included by default in the LAMP recipe.